TeachingRecordSystem/src/TeachingRecordSystem.Core/Models/InductionStatus.cs

```diff
@@ -35,6 +35,8 @@ public static class InductionStatusRegistry
 
     public static bool RequiresCompletedDate(this InductionStatus status) => _info[status].RequiresCompletedDate;
 
+    public static bool RequiresExemptionReason(this InductionStatus status) => _info[status].RequiresExemptionReason;
```

Comment:
```suggestion
    public static bool RequiresExemptionReasons(this InductionStatus status) => _info[status].RequiresExemptionReason;
```

Reply:
You want it to say RequiresExemptionReasons? but keep _info[status].RequiresExemptionReason;

Reply:
`Reasons` plural everywhere, ideally